src/chat.rs

```diff
@@ -420,7 +420,10 @@ impl ChatId {
         if changed {
             context.emit_event(EventType::MsgsChanged {
                 chat_id: self,
-                msg_id: MsgId::new(0),
+                msg_id: match self.get_draft_msg_id(context).await {
```

Comment:
This query is done even `msg` is `None` (we unset the draft). Could just return `MsgId::new(0)` without querying the database.